test/fx2trt/test_quant_trt.py

```diff
@@ -727,15 +727,18 @@ def forward(self, x, y):
         self.checkGraphModuleNodes(m.standalone, expected_node_occurrence=standalone_node_occurrence)
         m = _convert_fx_do_not_use(m, is_reference=True, backend_config_dict=backend_config_dict)
         node_occurrence = {
-            ns.call_function(torch.quantize_per_tensor): 3,
+            # two inputs for standalone module
+            ns.call_function(torch.quantize_per_tensor): 2,
```

Comment:
this is probably an error from previous PRs, not sure why we have 3 here before

Reply:
Did the test pass either way?